### ISSUE TITLE
Dependency parser evaluation atribute labels of gold object  return None  

### ISSUE THREAD

**Issue by sonyayahab** [explosion/spaCy#3567]
Currently I'm developing a new dependency parser model for Indonesia language. The model I had trained was fine. Now, I want to implement the evaluation function of the model. But, I got some error when do the evaluation, and the error have a correlation with the goldparse object.
## The code
```
import spacy
from nltk.metrics import ConfusionMatrix
from nltk.metrics.scores import precision, recall, f_measure, accuracy
from sklearn.metrics import accuracy_score, classification_report
from sklearn import metrics

from spacy.gold import GoldParse

path = '/home/theavenger/Documents/pipeline_annotation/dl/engine/dependency_parser/model/id'
model = spacy.load(path)

train_data = [('Di sektor ganda putri tidak ada wakil Indonesia yang lolos sedangkan di sektor tunggal putri Indonesia tidak mengirimkan wakilnya .', 
               {'heads': [5, 0, 1, 2, 5, 5, 5, 6, 9, 6, 17, 17, 11, 12, 13, 17, 17, 5, 17, 5], 
                'deps': ['prep', 'pobj', 'nn', 'nn', 'neg', 'root', 'dobj', 'nn', 'ref', 'rcmod', 'mark', 'prep', 'pobj', 'nn', 'nn', 'nsubj', 'neg', 'ccomp', 'dobj', 'punct'], 
                'words': ['Di', 'sektor', 'ganda', 'putri', 'tidak', 'ada', 'wakil', 'Indonesia', 'yang', 'lolos', 'sedangkan', 'di', 'sektor', 'tunggal', 'putri', 'Indonesia', 'tidak', 'mengirimkan', 'wakilnya', '.']}), 
              ('Dirinya akan fokus pada memperbaiki kelemahan - kelemahan yang masih terlihat jelas di Piala Presiden .', 
               {'heads': [2, 2, 2, 2, 3, 4, 5, 5, 10, 10, 5, 10, 10, 12, 13, 2], 
                'deps': ['nsubj', 'advmod', 'root', 'prep', 'pcomp', 'dobj', 'goeswith', 'goeswith', 'ref', 'advmod', 'rcmod', 'acomp', 'prep', 'pobj', 'flat', 'punct'], 
                'words': ['Dirinya', 'akan', 'fokus', 'pada', 'memperbaiki', 'kelemahan', '-', 'kelemahan', 'yang', 'masih', 'terlihat', 'jelas', 'di', 'Piala', 'Presiden', '.']})]

reference_train = []
predictions_train = []
for text, annot in train_data:
    doc = model(text)
    doc_gold_text = model.make_doc(text)
    gold = GoldParse(doc_gold_text, entities=annot['deps'])
    reference_train.extend([_ if _ is not None else 'X' for _ in gold.labels])
    predictions_train.extend([d.dep_ if d.dep_ is not None else 'X' for d in doc])
```
when I print the accuracy:
```
print('Accuracy: {}\n'.format(accuracy(reference_train, predictions_train)))
```
The accuracy I got is 0,0. Then I inspect the reference_train by print it, and as a comparison, I print also the predictions_train:
```
print(reference_train)
print(predictions_train)

##results:
['X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X', 'X']
['prep', 'pobj', 'nn', 'nn', 'neg', 'ROOT', 'dobj', 'nn', 'ref', 'rcmod', 'mark', 'prep', 'pobj', 'nn', 'nn', 'nsubj', 'neg', 'ccomp', 'dobj', 'punct', 'nsubj', 'advmod', 'ROOT', 'prep', 'pcomp', 'dobj', 'goeswith', 'goeswith', 'ref', 'advmod', 'rcmod', 'acomp', 'prep', 'pobj', 'flat', 'punct']
```
Before I'm using labels attribute, I try to call by using deps attribute, but there's none. So after I read the documentation at [GoldParse](https://spacy.io/api/goldparse), the attribute I think have to use is labels, but I got this error. What I'm trying to ask is, why there's no deps attribute at the gold object?. Anything suggestions for this problem will be helpful. 
Thank you.

## Your Environment
* Operating System: Ubuntu 18.10
* Python Version Used: Anaconda environment using python==3.6
* spaCy Version Used: 2.0

**Comment by honnibal** [comment]
Change:

```gold = GoldParse(doc_gold_text, entities=annot['deps'])```

To:

```gold = GoldParse(doc_gold_text, words=annot['words'], heads=annot['heads'], deps=annot['deps'])```

The `GoldParse` is putting the dependency labels in the named entities variable, because it's receiving the information on the wrong keyword argument.
